### ISSUE TITLE
Windows executable (v3.5.0 - 3.6.0) fails with `libstdc++-6.dll was not found`

### ISSUE THREAD

**Issue by DJPorv** [vale-cli/vale#863]
### Check for existing issues

- [X] Completed

### Environment

* OS: Windows
* Install method: scoop and direct download from gh releases
* Vale version: 3.5.0 and 3.6.0

### Describe the bug / provide steps to reproduce it

* Install vale with scoop (or download from releases and unzip)
* Run `vale --help`

Fails with two pop-ups:

_First pop-up_
```plaintext
---------------------------
vale.exe - System Error
---------------------------
The code execution cannot proceed because libstdc++-6.dll was not found. Reinstalling the program may fix this problem. 
---------------------------
OK   
---------------------------
```
_Second pop-up_
```plaintext
---------------------------
vale.exe - System Error
---------------------------
The code execution cannot proceed because libgcc_s_seh-1.dll was not found. Reinstalling the program may fix this problem. 
---------------------------
OK   
---------------------------
```

**Comment by sdwheeler** [comment]
Same here.

**Comment by jdkato** [comment]
This should be fixed now in version v3.6.1.

**Comment by sdwheeler** [comment]
I installed 3.6.1. That fixed the error. However, the version output is wrong.

```
PS> vale -v
vale version master
```
